RootTools/python/samples/samples_13TeV_RunIIFall17MiniAOD.py

```diff
@@ -307,19 +334,37 @@
 DiBosons = [
     WW,
     WWTo2L2Nu,
+    WWToLNuQQ,
+    WWToLNuQQ_ext,
     WW_DPS,
+    WW_DPSTo2L,
     WZ,
     WZTo3LNu_fxfx,
+    WZTo1L1Nu2Q,
     ZZ,
     ZZTo4L, ZZTo4L_ext,
     ZZTo2L2Nu,
 ]
 
 
+# ===  TRI-BOSONS
+
+# cross section from https://twiki.cern.ch/twiki/bin/view/CMS/SummaryTable1G25ns#Triboson
+WWW = kreator.makeMCComponent("WWW", "/WWW_4F_TuneCP5_13TeV-amcatnlo-pythia8/RunIIFall17MiniAOD-94X_mc2017_realistic_v11-v1/MINIAODSIM", "CMS", ".*root", 0.2086)
+WWZ = kreator.makeMCComponent("WWZ", "/WWZ_4F_TuneCP5_13TeV-amcatnlo-pythia8/RunIIFall17MiniAOD-94X_mc2017_realistic_v11-v1/MINIAODSIM", "CMS", ".*root",  0.1651 )
+WZZ = kreator.makeMCComponent("WZZ", "/WZZ_TuneCUETP8M1_13TeV-amcatnlo-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM", "CMS", ".*root",  0.05565 )
```

Comment:
should be `/WZZ_TuneCP5_13TeV-amcatnlo-pythia8/RunIIFall17MiniAOD-94X_mc2017_realistic_v11-v1/MINIAODSIM`  I suppose